lib/ddtrace/configuration.rb

```diff
@@ -56,7 +56,10 @@ def logger
     end
 
     def shutdown!
-      components.shutdown! if instance_variable_defined?(:@components) && @components
+      if instance_variable_defined?(:@components) && @components
```

Comment:
should this change be in a different PR?

Reply:
This is the important change, I updated the description of the PR to match.